cirq/experiments/fidelity_estimation.py

```diff
@@ -32,17 +33,21 @@
 
 import numpy as np
 import pandas as pd
+import scipy.optimize
+import sympy
 import tqdm
 
 from cirq import ops, protocols, sim
 from cirq.circuits import Circuit
+from cirq.google.calibration.phased_fsim import PhasedFSimCalibrationOptions
```

Comment:
I am trying to remove cirq.google dependencies from cirq core. Is this really necessary here? Should we pull up PhasedFSimCalibrationOptions to cirq core? PhasedFSimGate itself lives in cirq.google. Alternatively we can think about introducing cirq/google/experiments maybe?

Reply:
There is also another alternative to consider:
Make cirq.experiments.fidelity_estimation independent from the PhasedFSimCalibrationOptions and move the XEB options to the cirq.google.calibration. The XEB in cirq.google.calibration would then be a very thin wrapper on complicated functionality in the cirq.experiments.fidelity_estimation. It would only translate between options and estimation logic. I think this would be a good separation because in the end we'll have:
 1) XEB estimation in the cirq.google.experiments
 2) XEB estimation in the QE
 3) Floquet estimation in the QE
The interfaces of 1) and 2) are different and they would be unified in cirq.google.calibration.

There is not much effort do it like that actually: the variables from lines 757-761 would just need to be a method arguments and characterize_phased_fsim_parameters_with_xeb instead of options would need to take a simplex. I think it would be even cleaner that way.

All the options and dataclasses can be a subject of a follow-up PR. Hopefully, it would not clutter the notebook too much.

Reply:
Yeah, the inheritance from `PhasedFSimCalibrationOptions` isn't necessary for the `cirq.experiments` code or this PR. I wanted to foreshadow how it would be integrated into @mrwojtek 's cirq.google utilities, which has in some sense worked because the above comment goes through a good way for this to be done :)

Reply:
Great!

Reply:
removed